test/integration/host_js_test.rb

```diff
@@ -22,6 +22,37 @@ class HostJSTest < IntegrationTestWithJavascript
     SETTINGS[:organizations_enabled] = true
   end
 
+  describe 'multiple hosts selection' do
+    setup do
+      FactoryBot.create_list(:host, 2)
+    end
+
+    teardown do
+      Setting[:entries_per_page] = settings(:attributes16).value
```

Comment:
Please save the previous value in the setup and restore it in the teardown, so that it is clear what happens and it isn't dependent on a fixture